models/curvefi/ethereum/curvefi_ethereum_pool_details.sql

```diff
@@ -637,18 +637,18 @@ FROM
                 'Regular',
                 'susdv2',
                 'susd',
-                '0xA5407eAE9Ba41422680e2e00537571bcC53efBfd',
-                '0xC25a3A3b969415c80451098fa907EC722572917f',
-                '0xA90996896660DEcC6E997655E065b23788857849',
-                '0x6b175474e89094c44da98b954eedeac495271d0f',
-                '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-                '0xdac17f958d2ee523a2206206994597c13d831ec7',
-                '0x6b175474e89094c44da98b954eedeac495271d0f',
-                '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-                '0xdac17f958d2ee523a2206206994597c13d831ec7',
-                '0x57Ab1E02fEE23774580C119740129eAC7081e9D3',
-                '0xFCBa3E75865d2d561BE8D220616520c171F12851',
-                '0x57ab1ec28d129707052df4df418d58a2d46d5f51'
+                '0xA5407eAE9Ba41422680e2e00537571bcC53efBfd', --pool_address
```

Comment:
common theme i see on spark: lowercase address -- should we put a lower() function in the select statement at the top? or are these fine as-is?